src/server/raft_client.rs

```diff
@@ -853,6 +881,14 @@ async fn start<S, R, E>(
                     .router
                     .broadcast_unreachable(back_end.store_id);
             }
+            Ok(RaftCallRes::Closed) | Err(_) => {
+                error!("connection close"; "store_id" => back_end.store_id, "addr" => addr);
```

Comment:
Why? It should retry infinitely until the store is tombstone.

Reply:
If it is closed actively, it's different from aborted, we don't need to retry it.

Reply:
The RPC is an infinite RPC. If it's closed actively, there should be either bugs or error loss. Either cases we should always retry.

Reply:
revert it